frontend/src/main/pages/PlayPage.js

```diff
@@ -88,10 +87,8 @@ export default function PlayPage() {
     mutationbuy.mutate(userCommons)
   };
 
-
-  const onSuccessSell = () => {
-    toast(`Cow sold!`);
-  }
+const onSuccessSell = () => {
```

Comment:
if this function is empty is there a way you could get rid of it entirely? seems unnecessary to have it if there's no toast

Reply:
Andrew raises a good point.  I'm ok with leaving it so that if there is anything in the future we want to do when the cow sale is successful than we have a place to put it, but strictly speaking it isn't necessary, and wouldn't hurt to remove it.